### DISCUSSION TITLE
Bagaimana Cara Memperbesar plot di rstudio

### DISCUSSION THREAD

**Discussion by ghost** [teamalgoritma/community#82] [Q&A] [answered]
Halo,saya ingin bertanya bagaimana ya cara memperlebar plot di rstudio ya?

**Comment by onlyphantom** [comment, marked as answer]
If you're using R Markdown in RStudio and want to have a wider plot, use the `fig.width` and `fig.height` (in inches) option. Pass them in along with other parameters for the chunk option

For example this makes the plot 7 inches wide, and 3 inches tall

```
```{r message=FALSE, warning=FALSE, fig.width=7, fig.height=3}
ggplot(df, 
      aes(x=Time, 
          y=value,
          fill=variable)) +
  geom_col(position = "dodge") 
```
```

**Reply by ghost** [reply]
Ooo pakai fig.width and fig.height ya thanks , tapi btw sepertinya yg di example itu ngga kecopy bagian fig.wedth / fig.height nya ya?
"ggplot(df, 
      aes(x=Time, 
          y=value,
          fill=variable)) +
  geom_col(position = "dodge") " saja

**Reply by onlyphantom** [reply]
Sorry! Updated!
